stun_server/region/module.tf

```diff
@@ -0,0 +1,42 @@
+locals {
```

Comment:
`data.tfe_outputs.infrastructure.values[data.aws_region.current.name]` is reused quite a fair bit, might be a contender to be a local as `infrastructure_regioin`

Reply:
Made a local `infrastructure_region_outputs`.